### DISCUSSION TITLE
Is this going to be updated to deal with the new encryption or support ending?

### DISCUSSION THREAD

**Discussion by Icezman001** [Akandesh/BlazeDumper-Rust#14] [General]
See title

**Comment by Akandesh** [comment]
I don't know, currently taking a break from it but I think it's doable with some signature work, lacking motivation right now though.

**Reply by Icezman001** [reply]
Yeah I understand, it's something that gets done in a 1 week frenzy when you're motivated.

We should try to pool some people together on UC or have some people do some pull requests with maybe a way to detect encryption methods through signature/pattern scan and then converting the psuedocode from the function in IDA to working code, which can then be called.

Guessing it won't be as easy as before but could perhaps be a standalone C++ or python program (that opens IDA and gets the correct function and turns it into working code for c++).

Not impossible, just annoying.

Also i don't know if the encryption function changes each time or not?
